src/python/grpcio/grpc/_channel.py

```diff
@@ -1439,15 +1588,19 @@ def _unsubscribe(state, callback):
                 break
 
 
-def _augment_options(base_options, compression):
+def _augment_options(
+        base_options: Optional[ChannelArgumentType],
```

Comment:
I'm realizing now that this name is misleading. `ChannelArgument` is singular, but this is actually of type `Sequence[Tuple[str, Any]]`. Can we make `ChannelArgumentType` just be `Tuple[str, Any]`? This way, we will see `Sequence[ChannelArgumentType]` in `_channel.py`.

Reply:
Sure that's sounds more clear, will change `ChannelArgumentType` to `Tuple[str, Any]`